topbeat/etc/topbeat.template.json

```diff
@@ -36,6 +36,22 @@
             }
           }
         },
+        "cpus": {
```

Comment:
@monicasarbu This changes were added when I run the new script. I assume it already wasn't up-to-date before. Will this have an affect?